source/Core/Controller/BaseController.php

```diff
@@ -268,7 +275,7 @@ public function addGlobalParams($oShop = null)
     /**
      * Sets value to parameter used by template engine.
      *
-     * @param string $sPara  name of parameter to pass
+     * @param mixed  $sPara  name of parameter to pass
```

Comment:
In that line above I guess the function parameter type `sting` in the comment was correct for the parameter name but needs to be changed for the value

Reply:
corrected in https://github.com/OXID-eSales/oxideshop_ce/pull/676/commits/5eab81d2860e99f5db83cd83e1afc94c1e78550d